docs/concepts/tasks.md

```diff
@@ -204,29 +204,80 @@ def my_flow():
 
 ## Retries
 
-Prefect tasks can automatically retry on failure. To enable retries, pass `retries` and `retry_delay_seconds` parameters to your task.
+Prefect can automatically retry tasks on failure. In Prefect, a task _fails_ 
+if it returns a value that is not a `State` object, or if it raises an
```

Comment:
```suggestion
if its python function raises an exception or, uncommonly, returns a value that is not a `State` object.
```

Reply:
@billpalombi Isn't a task a function? 😅

Reply:
Actually I see what you mean now

Reply:
I'm going to leave out the `State` part for now because it doesn't make sense and I don't understand what would cause that. (It can't be that the user-defined function returned an object other than a `State`...).